src/missionwaypointmsg.py

```diff
@@ -0,0 +1,27 @@
+import roslibpy
+import numpy as np
+from drone import Drone
+from abc import ABC, abstractmethod
+from enum import Enum
+
+class MissionWaypointMsg(ABC):
+
```

Comment:
What does this script do?

Reply:
After searching, there are no external references or imports to this file. It appears it was created in order to provide more specific parameters for each waypoint, however some of these values are not necessary (like yaw) or are hardware specific (like gimbal_pitch). If there are specific functions or properties of a drone that needs to be controlled at a waypoint, it is better that it is taken care of in the specific drone instance itself. I recommend this is terminated.